common/views/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -53,17 +54,16 @@ const Breadcrumb: FunctionComponent<BreadcrumbItems> = ({ items }) => {
             >
               {text}
             </LinkOrSpanTag>
-          </ItemWrapper>
+          </Space>
         );
       })}
-      {/* Because we always insert a 'Home' breadcrumb, we know it will be non-empty. */}
       <script
         type="application/ld+json"
         dangerouslySetInnerHTML={{
-          __html: JSON.stringify(breadcrumbsLd({ items })),
+          __html: JSON.stringify(breadcrumbsLd({ items: visibleItems })),
```

Comment:
I'm not sure – just because we don't display the breadcrumb visually, does it necessarily mean we don't want to include it for the JSON-LD structure? I have a vague sense that we discussed this long ago, but I could be wrong. I'll see if I can find anything

Reply:
Can't find anything – maybe it was a conversation. Or I imagined it.

Reply:
What you says make sense though, I will to revert. It just means that "Home" is not part of the list, which is probably fine.

Reply:
Now reverted, so will merge, thank you!